src/components/TitleHashButton.tsx

```diff
@@ -80,3 +98,31 @@ export default function HashButtonCopyable({hash}: HashButtonCopyableProps) {
     </Box>
   );
 }
+
+function Name({address}: {address: string}) {
+  const theme = useTheme();
+  const name = useGetNameFromAddress(address);
+
+  if (!name) {
+    return null;
+  }
+
+  return (
+    <Box>
+      <Stack
+        justifyContent="center"
+        sx={{
+          height: BUTTON_HEIGHT,
+          backgroundColor: `${
+            theme.palette.mode === "dark" ? grey[600] : grey[200]
+          }`,
+          borderRadius: 1,
+          color: "inherit",
+          padding: "0.15rem 1rem 0.15rem 1rem",
+        }}
+      >
+        <Typography variant="body2">{name}</Typography>
```

Comment:
should we show `.apt`? `{label}.apt`

Reply:
I would argue that it's not needed? For example etherscan does not show ".eth". Also I'm following Petra who shows the name only.